docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1137 @@
+{
```

Comment:
<p>Interesting. So the second term in <code>alpha</code> has no effect on the partial derivative (i.e., only the first term matters).</p><p> </p><p>Can you please explain why the sum of <code>alpha</code> is zero? I see that it is zero in the other partial derivative but what does it have to do with this partial derivative? Btw, <code>mu^prime(0)</code> looks as if you are evaluating <code>mu^prime</code> as a function. Instead, you may want to use a <code>\cdot</code> instead to show that it is a multiplication </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
> Interesting. So the second term in `alpha` has no effect on the partial derivative (i.e., only the first term matters).

Yes, that's right. In `\alpha`, the variables '\mu_prime` and `\sigma_prime` appear only in the first term. So, when we take derivatives w.r.t to these variables, the second term of `\alpha` becomes 0.

> Can you please explain why the sum of `alpha` is zero? I see that it is zero in the other partial derivative but what does it have to do with this partial derivative?

Please note that  (1) and (2) are a single system of equations. That is, we want to find `\mu_prime` and `sigma_prime` such that they satisfy both (1) and (2). So, if we get to an equation derived from (1) or (2), the equation should hold true in the other one. So, after working with (1), we realized `\sum\alpha_prime = 0` (at the critical point we are looking for). That means the values of `\mu_prime` and `sigma_prime` should satisfy equation (3). In other words, whenever we see `\sum\mu_prime` during our solution, we can replace it with 0. Please let me know  if my answer is not clear.



> Btw, `mu^prime(0)` looks as if you are evaluating `mu^prime` as a function. Instead, you may want to use a `\cdot` instead to show that it is a multiplication

Correct! I will change it in the next set of commits. Thanks!

Reply:
<blockquote>Please note that (1) and (2) are a single system of equations. Please let me know if my answer is not clear.</blockquote><p> </p><p>Understood. You'll have to forgive me as it's been a long time since I've had to think about this :) It would be helpful to add this reminder to the notebook</p>

Reply:
Sure...I will add a note regarding this matter.

Reply:
<p>I was just thinking about this a little bit more and your math/engineering skills are really shining through here and I get to learn from you too! So, thank you. You've reminded me that, conceptually, we are looking for the global minima in 2-dimensional space and hence why the partial derivatives form a system of equations. This is fun :)</p>

Reply:
It IS fun indeed! 

> conceptually, we are looking for the global minima in 2-dimensional space and hence why the partial derivatives form a system of equations.

Exactly!!!